### PR TITLE
Explicitly ban unsafe package for serialization

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java [hunk 1]
```diff
@@ -190,6 +192,22 @@ private Class<?> doLookup(String name) {
         }
     }
 
+    private boolean banned(String name) {
+        if (name.isEmpty())
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java:196]: why would name be empty?

Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java:196]: It shouldn't, that's why it's banned

Comment (JerryShea) [reply src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java:196]: should we throw an exception or assert instead?

**Review by JerryShea — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java (hunk 1 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java:196]: I could leave this code out, but you would get an IndexOutOfBoundException which could be confusing.

Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java:196]: All banned class names result in a specific ClassNotFoundRuntimeException

**Pull request merged**